recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -25,10 +25,20 @@ if "%CI%" == "" (
     echo - %CI% >> .ci_support\%CONFIG%.yaml
 )
 
-:: Remove chocolatey gfortran.
-if "%CI%" == "azure" (
-  rm "C:\ProgramData\Chocolatey\bin\gfortran.exe"
-)
+:: Remove some directories from PATH
+set "PATH=%PATH:C:\ProgramData\Chocolatey\bin;=%"
+set "PATH=%PATH:C:\Program Files (x86)\sbt\bin;=%"
+set "PATH=%PATH:C:\Rust\.cargo\bin;=%"
+set "PATH=%PATH:C:\Program Files\Git\usr\bin;=%"
+set "PATH=%PATH:C:\Program Files\Git\cmd;=%"
+set "PATH=%PATH:C:\Program Files\Git\mingw64\bin;=%"
+set "PATH=%PATH:C:\Program Files (x86)\Subversion\bin;=%"
+set "PATH=%PATH:C:\Program Files\CMake\bin;=%"
+set "PATH=%PATH:C:\Program Files\OpenSSL\bin;=%"
+set "PATH=%PATH:C:\Strawberry\c\bin;=%"
+set "PATH=%PATH:C:\Strawberry\perl\bin;=%"
+set "PATH=%PATH:C:\Strawberry\perl\site\bin;=%"
+set "PATH=%PATH:c:\tools\php;=%"
```

Comment:
AppVeyor was cleaner on this regard. Lots of pre-installed things, little interference.

Reply:
Appveyor had lots of things pre-installed, but they didn't add those to PATH.

Reply:
Yeah. That is what I meant with my rant above. AppVeyor have tons of experience on Windows stuff that Azure could use ;-p